terraform/shared/services/ecs-cluster/main.tf

```diff
@@ -14,8 +14,7 @@ locals {
 }
 
 resource "aws_ecs_cluster" "cluster" {
-  name               = var.cluster_name
-  capacity_providers = ["FARGATE", "FARGATE_SPOT"]
```

Comment:
the version upgrade [guide](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/version-5-upgrade) suggest to remove this field.

<img width="688" alt="image" src="https://github.com/user-attachments/assets/a530f84e-ecfe-423a-aed1-c6889f9eac1f">